### PR TITLE
fix(Cloud integrations): Uncapitalized a word in the doc title

### PR DESCRIPTION
**Pull request by paperclypse**
Fixed while writing release notes

### CODE REVIEW

File: src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Dbt Cloud with Airflow and Snowflake Integration 
+title: Dbt Cloud with Airflow and Snowflake integration
```
Comment (rhetoric101) [inline src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx:2]: @paperclypse Should this be "dbt" (lower case)? The product name appears to be lower case in their [docs](https://www.getdbt.com/product/what-is-dbt) and we use "dbt" in the doc itself.

**Review by rhetoric101 — COMMENTED** [review]
@paperclypse I just had one question.

File: src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx (hunk 1 above)
Comment (paperclypse) [inline src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx:2]:
```suggestion
title: dbt Cloud with Airflow and Snowflake integration 
```

**Review by rhetoric101 — APPROVED** [review]
@paperclypse this looks good!

**Pull request merged**